cypress/integration/pages/articles/testsForAMPOnly.js

```diff
@@ -55,33 +57,18 @@ export const testsThatFollowSmokeTestConfigForAMPOnly = ({
           });
         });
 
-        // Tests requiring iframe access are temporarily being throttled to the 'news' service.
-        if (service === 'news') {
-          it('should autoplay', () => {
-            cy.request(`${Cypress.env('currentPath')}.json`).then(
-              ({ body }) => {
-                const media = getBlockData('video', body);
-                if (media && media.type === 'video') {
-                  cy.get(
-                    'div[class^="StyledVideoContainer"] iframe[class^="i-amphtml-fill-content"]',
-                  )
-                    .scrollIntoView()
-                    .then($iframe => {
-                      cy.wrap($iframe.prop('contentWindow'), {
-                        // `timeout` only applies to the methods chained below.
-                        // `its()` benefits from this, and will wait up to 8s
-                        // for the mediaPlayer instance to become available.
-                        timeout: 20000,
-                      })
-                        .its('embeddedMedia.playerInstances.mediaPlayer')
-                        .invoke('currentTime')
-                        .should('be.gt', 0);
-                    });
-                }
-              },
-            );
+        it('should render an iframe with a valid URL when a user clicks play', () => {
+          cy.request(`${Cypress.env('currentPath')}.json`).then(({ body }) => {
+            const media = getBlockData('video', body);
+
+            if (media && media.type === 'video') {
+              const { lang } = appConfig[service][variant];
+              const embedUrl = `${getVideoEmbedUrl(body, lang)}/amp`;
+              cy.testResponseCodeAndType(embedUrl, 200, 'text/html');
+              cy.get(`amp-iframe[src="${embedUrl}"]`).should('be.visible');
```

Comment:
Would it make sense to swap these lines - check if the iframe is visible first, before checking the endpoint?

Reply:
Yes, I think it probably would - I'll swap them!

Reply:
Thanks :)